@xen-orchestra/lite/src/components/ui/UiButton.vue

```diff
@@ -24,10 +24,12 @@ const props = withDefaults(
     busy?: boolean;
     disabled?: boolean;
     icon?: IconDefinition;
-    color?: "primary" | "secondary";
+    color?: "info" | "error" | "warning" | "success";
```

Comment:
What do you think of the creation of a `Colors` type in order to avoid code duplication since it may be often reused.
```ts
export type Colors = "info" | "error" | "warning" | "success"
```